### ISSUE TITLE
Multiselect: aria attributes should not be set on <li> elements

### ISSUE THREAD

**Issue by markuczy** [primefaces/primeng#14946]
### Describe the bug

<li> elements for each multiselect item should not have the following properties set:
- aria-label
- aria-setsize
- aria-posinset
- aria-selected

Screenshot of the accessibility checker report taken from the documentation(https://primeng.org/multiselect):
![image](https://github.com/primefaces/primeng/assets/129275100/7110ddd1-1fe9-4cbb-aa7a-82dc36cfbce0)

### Environment

Ubuntu 22.04, WSL 2, Windows 11 PRO

### Angular version

15.2.10

### PrimeNG version

17.9.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

### Expected behavior

Expected Multiselect component to not raise any accessibility issues.

**Comment by vinodneppalli62** [comment]
Each li tag is missing role='option' which is why probably aria checker is complaining.
